std_msgs/Header header
shape_msgs/Mesh[] meshes